src/py/_pyodide/_core_docs.py

```diff
@@ -909,19 +954,70 @@ def create_proxy(
 # from python2js
 
 
+@overload
+def to_js(
+    obj: list[Any] | tuple[Any],
```

Comment:
It's unfortunate we have to repeat the rest of the annotations, just for this argument and output type, but I also don't see a way around it.